### ISSUE TITLE
Moloch hearts need to have bedrock-level hardness

### ISSUE THREAD

**Issue by ramou** [Concordia-Modding-Community/moloch#18]
Or anything else that would make them immovable/unbreakable

**Comment by ramou** [comment]
Something like:
```
register("bedrock", new Block(AbstractBlock.Properties.create(Material.ROCK).hardnessAndResistance(-1.0F, 3600000.0F).noDrops().setAllowsSpawn(Blocks::neverAllowSpawn)))
```

**Comment by alexandre-lavoie** [comment]
Fixed in 2a378f7
